engines/ultima/nuvie/metaengine.cpp

```diff
@@ -56,133 +56,133 @@ struct NuvieActionDescription {
-	{ "QUICK_LOAD", "Quick load", nullptr, nullptr },
-	{ "QUIT", "Quit", "q", "C+q" },
-	//{ "QUIT_NO_DIALOG", "Quit without confirmation", "M+q", nullptr },
-	{ "GAME_MENU_DIALOG", "Show game menu; Cancel action if in middle of action", "ESCAPE", nullptr },
-	//{ "TOGGLE_FULLSCREEN", "Toggle Fullscreen", "M+RETURN", nullptr },
-	{ "TOGGLE_CURSOR", "Toggle Cursor", "TAB", nullptr },
-	{ "TOGGLE_COMBAT_STRATEGY", "Toggle combat strategy", "BACKQUOTE", nullptr },
-	{ "TOGGLE_FPS_DISPLAY", "Toggle frames per second display", "C+PLUS", nullptr },
-	{ "TOGGLE_AUDIO", "Toggle audio", "C+a", nullptr },
-	{ "TOGGLE_MUSIC", "Toggle music", "C+m", nullptr },
-	{ "TOGGLE_SFX", "Toggle sfx", "C+s", nullptr },
-	{ "TOGGLE_ORIGINAL_STYLE_COMMAND_BAR", "Show/hide original style command bar", nullptr, nullptr },
-	{ "DO_ACTION", "Do action", "RETURN", "KP_ENTER" },
-	{ "CANCEL_ACTION", "Cancel action", "SPACE", nullptr },
-	{ "MSG_SCROLL_UP", "Msg scroll up", "PAGEUP", nullptr },
-	{ "MSG_SCROLL_DOWN", "Msg scroll down", "PAGEDOWN", nullptr },
-	{ "SHOW_KEYS", "Show keys", "h", nullptr },
-	{ "DECREASE_DEBUG", "Decrease debug", "C+d", nullptr },
-	{ "INCREASE_DEBUG", "Increase debug", "C+i", nullptr },
-	{ "CLOSE_GUMPS", "Close gumps", "z", nullptr },
-	//{ "USE_ITEM", "Use item", nullptr, nullptr }, // TODO: this takes an item no parameter
-	{ "TOGGLE_ALT_CODE_MODE", "Enter ALT code (hold)", "LALT", nullptr },
-	{ "ALT_CODE_0", "ALT Code 0", "A+KP0", "A+0" },
-	{ "ALT_CODE_1", "ALT Code 1", "A+KP1", "A+1" },
-	{ "ALT_CODE_2", "ALT Code 2", "A+KP2", "A+2" },
-	{ "ALT_CODE_3", "ALT Code 3", "A+KP3", "A+3" },
-	{ "ALT_CODE_4", "ALT Code 4", "A+KP4", "A+4" },
-	{ "ALT_CODE_5", "ALT Code 5", "A+KP5", "A+5" },
-	{ "ALT_CODE_6", "ALT Code 6", "A+KP6", "A+6" },
-	{ "ALT_CODE_7", "ALT Code 7", "A+KP7", "A+7" },
-	{ "ALT_CODE_8", "ALT Code 8", "A+KP8", "A+8" },
-	{ "ALT_CODE_9", "ALT Code 9", "A+KP9", "A+9" },
+	{ "WALK_WEST", _s("Walk west"), "LEFT", "KP4" },
+	{ "WALK_EAST", _s("Walk east"), "RIGHT", "KP6" },
+	{ "WALK_NORTH", _s("Walk north"), "UP", "KP8" },
+	{ "WALK_SOUTH", _s("Walk south"), "DOWN", "KP2" },
+	{ "WALK_NORTH_EAST", _s("Walk north-east"), "KP9", nullptr },
+	{ "WALK_SOUTH_EAST", _s("Walk south-east"), "KP3", nullptr },
+	{ "WALK_NORTH_WEST", _s("Walk north-west"), "KP7", nullptr },
+	{ "WALK_SOUTH_WEST", _s("Walk south-west"), "KP1", nullptr },
+	{ "LOOK", _s("Look"), "l", nullptr },
+	{ "TALK", _s("Talk"), "t", nullptr },
+	{ "USE", _s("Use"), "u", nullptr },
+	{ "GET", _s("Get"), "g", nullptr },
+	{ "MOVE", _s("Move"), "m", nullptr },
+	{ "DROP", _s("Drop"), "d", nullptr },
+	{ "TOGGLE_COMBAT", _s("Toggle combat"), "b", nullptr },
+	{ "ATTACK", _s("Attack"), "a", nullptr },
+	{ "MULTI_USE", _s("Multi-use"), "SEMICOLON", nullptr },
+	{ "SELECT_COMMAND_BAR", _s("Select command bar"), nullptr, nullptr },
+	{ "NEW_COMMAND_BAR", _s("Select new command bar"), "COMMA", nullptr },
+	{ "DOLL_GUMP", _s("Doll gump"), "i", nullptr },
```

Comment:
We should add this comment above this line and similar "gump" related descriptions:
// I18N: gump is Graphical User Menu Pop-up